api/src/main/java/org/eclipse/microprofile/config/spi/Converter.java

```diff
@@ -24,7 +24,7 @@
  * <p>A Converter can specify a {@link javax.annotation.Priority}.
```

Comment:
For supporting @Priority, we should include common annotations JSR (JSR-250) into MP, thoughts?